### DISCUSSION TITLE
Tau crashes

### DISCUSSION THREAD

**Discussion by AchmodinIVSWE** [ppy/osu#17720] [Q&A] [answered]
when I start a beatmap with Tau the game crashes when gameplay starts. and it only happens with tau. and I don't know why it suddenly happened because it worked fine before

edit: it seems it only happens for some maps and others it works.

**Comment by peppy** [comment, marked as answer]
Please report this to the Tau developer.

**Comment by naoei** [comment]
Seems like we still haven't received an issue ticket. If you are confused on where to create the issue, please do so over here: [https://github.com/taulazer/tau/issues/new](https://github.com/taulazer/tau/issues/new). Please also include your log files and beatmaps that is causing this issue.

We've had a very similar issue created not too long ago, but I just want to make sure that if it isn't the same, we have an issue created for it on our repository.
